redaxo/src/addons/be_style/package.yml

```diff
@@ -6,3 +6,6 @@ supportpage: www.redaxo.org/de/forum/
 load: early
 system_plugins: [redaxo]
 compile: 0
+
+console_commands:
+    be_style:compile: rex_be_style_command_compile
```

Comment:
haben wir sonst newlines am ende?

Reply:
Ja, bei php-dateien fordert das auch der cs check. aber für yml haben wir ja keine cs checks.
@bloep würde empfehlen, deinen editor so einzustellen, dass er das automatisch immer einfügt.

Reply:
in den anderen Addons sind welche, hier waren komischerweise anscheinend keine. Würde ich aber eben anfügen.

Reply:
ist angefügt, danke für den Tipp, dass im Editor einzustellen.

Reply:
Bin mir beim Namen unsicher. Üblicher ist bei den Commands Bindestriche statt Unterstriche zu nutzen.
So haben wir schon `user:set-password` und `db:connection-options`.
Allerdings andererseits ist `be_style` nun mal der Name des Addons, und nicht etwa `be-style`.
Was meint ihr? `be_style:compile`, `be-style:compile`, oder noch was anderes?

Reply:
Wie wäre es mit `sass:compile` ?

Reply:
eigentlich würde ich schon sagen eher `be_style` als `be-style`.
`sass:compile` finde ich auch nicht schlecht. Da hätte ich dann aber gerne einen alias `scss:compile`.

Reply:
> Da hätte ich dann aber gerne einen alias scss:compile.

Dann vielleicht `styles:compile`. Das hätte dann sogar vom Wortlaut her einen Bezug auf die Gulptasks vom [Bimmelbam](https://github.com/FriendsOfREDAXO/redaxo-mit-bimmelbam/blob/master/gulpfile.js/config.js#L112).

Wer weiß was man noch alles später kompilieren könnte (javascript, svgs).
Alternativen: `compile:styles` oder `task:styles`

Reply:
Nochmal drüber nachgedacht: Ich bin für entweder doch bei `be_style:compile` bleiben, oder für `styles:compile`.

Reply:
`styles:compile` 👍

Reply:
Wir hatten soweit ich mich erinnere bisher ein namensschema was an erster stelle den addon-namen vorsieht. Das sollten wir IMO beibehalten.

Man könnte einen alias zusätzlich registrieren der dann vom schema abweicht

Reply:
ich würde vorschlagen `be_style:compile` als primären (wegen einheitliches namensschema) und `styles:compile` als alias.

Reply:
Würde in der Übersicht so aussehen, wenn mit `->setAliases(['styles:compile'])` festgelegt.
![image](https://user-images.githubusercontent.com/21292755/43034523-3ddfc7d4-8cde-11e8-96bd-d01e0e9ea1f8.png)

Reply:
> Wir hatten soweit ich mich erinnere bisher ein namensschema was an erster stelle den addon-namen vorsieht. Das sollten wir IMO beibehalten.

Mein Stand ist, dass wir gesagt haben, das Addon muss nicht unbedingt an erster Stelle stehen, sondern es soll ein ins Gesamtschema passender Name gewählt werden.
So soll das cache_warmup-Addon zum Beispiel den Command `cache:warmup` (passend zu `cache:clear` registrieren, und nicht etwa `cache_warmup:warmup` oder so.
Trotzdem kann es oftmals sinnvoll sein, den Addonnamen als Präfix zu nehmen. Aber es ist keine starre Regel.

Ich persönlich würde keinen Alias definieren, sondern es auf einen Namen beschränken. 
Finde, ein solcher Alias bringt kaum Mehrwert, eher Verwirrung.
Welcher Name es dann würde, `be_style:compile` oder `styles:compile`, ist mir egal.
(Lasse mich aber selbstverständlich auch überstimmen, wenn ihr beide Namen haben wollt)

Reply:
wie wollen wir hier verbleiben ? ich finde den alias jetzt nicht tragisch.